### PR TITLE
Guide feature proposals to the product discussions space

### PR DESCRIPTION
**Pull request by gmourier**
☝️

### CODE REVIEW

**Comment by curquiza** [comment]
Hum, I was thinking: is it something you want to tell only to the contributors, or in priority to the users (so in the README)?

**Comment by gmourier** [comment]
I'm thinking about the potential people who come to this repo. 

We wanted to avoid them creating an issue here asking for a feature and guide them as best as possible in the product discussions space.

We could also add this message when creating an issue, as we did on the meilisearch repo. WDYT?


Capture d’écran 2022-05-05 à 13 36 05

**Comment by curquiza** [comment]
yes issue template is one of the solutions!

**Comment by ManyTheFish** [comment]
let's do both, thanks @gmourier

**Review by curquiza — DISMISSED** [review]
Thanks guigui!!

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+contact_links:
+  - name: Feature request & feedback
+    url: https://github.com/meilisearch/product/discussions/categories/feedback-feature-proposal
```
Comment (ManyTheFish) [inline .github/ISSUE_TEMPLATE/config.yml:3]: We should redirect to the specialized label, no?
```suggestion
    url: https://github.com/meilisearch/product/discussions?discussions_q=label%3Aproduct%3Acore%3Atokenizer
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+    about: The feature requests and feedback regarding the already existing features are not managed in this repository. Please open a discussion in our dedicated product repository.
```
Comment (ManyTheFish) [inline .github/ISSUE_TEMPLATE/config.yml:4]:
```suggestion
    about: The feature requests and feedback like language support are not managed in this repository. Please open a discussion in our dedicated product repository.
```

**Review by ManyTheFish — CHANGES_REQUESTED** [review]

**Review by ManyTheFish — APPROVED** [review]
Thank you @gmourier!
bors merge

**Pull request merged**